src/matrix.jl

```diff
@@ -165,28 +198,66 @@ function ∈(M::AbstractMatrix, A::AbstractIntervalMatrix)
     return true
 end
 
+# random interval
+function rand(::Type{Interval}; N::Type{<:Real}=Float64,
```

Comment:
OK; i was thinking about adding a new `constructor.jl` file and then `operations.jl` with the stuff needed; `rand(::Type{Interval}` is only needed in `rand(::Type{IntevalMatrix})` and it can be just an internal function since it is not meant to be used for outside, like `_rand(::Type{Interval}..)`.

Reply:
`constructor.jl` with one method? I do not think this is useful.